caas/start_android_qcow2.sh

```diff
@@ -59,6 +59,7 @@ common_options="\
  -audiodev id=android_spk,timer-period=5000,driver=pa \
  -drive file=$caas_image,if=none,id=disk1 \
  -device virtio-blk-pci,drive=disk1,bootindex=1 \
+ -device vhost-vsock-pci,id=vhost-vsock-pci0,guest-cid=3 \
```

Comment:
In the commit message, can you explain why guest-cid is 3?

Reply:
done